src/components/AddBookForm.js

```diff
@@ -1,11 +1,28 @@
-import React from 'react';
+/* eslint-disable react/prop-types */
+import React, { useState } from 'react';
 
-const AddBookForm = () => (
-  <form>
-    <input type="text" name="Title" placeholder="Title" />
-    <input type="text" name="Author" placeholder="Author" />
-    <input type="button" value="Add" />
-  </form>
-);
+export default function Form(props) {
+  const [book, setBook] = useState({
+    title: '',
+    author: '',
+  });
 
-export default AddBookForm;
+  const onChange = (e) => {
+    if (book[e.target.name] !== e.target.value) {
+      setBook({
+        ...book,
+        [e.target.name]: e.target.value,
+      });
+    }
+  };
+
+  const { add } = props;
+
+  return (
+    <form>
+      <input onChange={onChange} type="text" name="title" placeholder="Title" />
+      <input onChange={onChange} type="text" name="author" placeholder="Author" />
+      <button type="button" onClick={() => add(book)}>Add Book</button>
+    </form>
+  );
+}
```

Comment:
- [ ] Great job you are adding the book however let us make sure that a user shouldn't add an empty data.

  - Validate the input fields, Don't allow users to add the empty data 
![cpp2](https://user-images.githubusercontent.com/55635977/132890387-0fe982fc-89c9-4133-bb40-565632169a0c.png)

- [ ] After adding a book info the input field should be empty so that user should add another book.
  

![cpp3](https://user-images.githubusercontent.com/55635977/132890394-bd7a77c7-f7ec-45e2-8c9c-8ee80ab174bb.png)

### Optional

- the categories link should display the `under construction` message.
![cpp1](https://user-images.githubusercontent.com/55635977/132890355-cf3e1364-9e50-4fc9-b5e3-e61c2c434d79.png)